### PR TITLE
[example] support stable diffusion v2

### PR DESCRIPTION
**Pull request by Fazziekey**
- update stable diffusion to v2 from(https://github.com/Fazziekey/stablediffusion)

### CODE REVIEW

File: examples/images/diffusion/README.md [hunk 1]
```diff
@@ -39,9 +39,17 @@ pip install transformers==4.19.2 diffusers invisible-watermark
...
+git clone https://github.com/1SAA/lightning.git
+git checkout strategy/colossalai
+export PACKAGE_NAME=pytorch
+pip install .
+```
+
 ### Install [Colossal-AI v0.1.10](https://colossalai.org/download/) From Our Official Website
 ```
-pip install colossalai==0.1.10+torch1.11cu11.3 -f https://release.colossalai.org
+pip install colossalai==0.1.11rc5+torch1.12cu11.3 -f https://release.colossalai.org
```
Comment (FrankLeeeee) [inline examples/images/diffusion/README.md:52]: Should it be `0.1.12` now?

File: examples/images/diffusion/configs/train_colossalai_teyvat.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 model:
-  base_learning_rate: 1.0e-04
+  base_learning_rate: 1.0e-4
   target: ldm.models.diffusion.ddpm.LatentDiffusion
   params:
+    parameterization: "v"
     linear_start: 0.00085
```
Comment (FrankLeeeee) [inline examples/images/diffusion/configs/train_colossalai_teyvat.yaml:6]: Add a comment for the meaning of `parameterization: v`

File: examples/images/diffusion/README.md (hunk 1 above)
Comment (Fazziekey) [reply examples/images/diffusion/README.md:52]: I want to provide a stable version of colossalai for users, which will not cause conflict

File: examples/images/diffusion/ldm/util.py [hunk 3]
```diff
@@ -45,7 +39,7 @@ def ismap(x):
 
 
 def isimage(x):
-    if not isinstance(x, torch.Tensor):
+    if not isinstance(x,torch.Tensor):
         return False
```
Comment (FrankLeeeee) [inline examples/images/diffusion/ldm/util.py:43]: Did you use `pre-commit`? This file seems to contain mainly format change. `pre-commit` should format the code in a way that there is a space between `x` and `torch.Tensor`.

File: examples/images/diffusion/configs/train_colossalai_teyvat.yaml (hunk 2 above)
Comment (Fazziekey) [reply examples/images/diffusion/configs/train_colossalai_teyvat.yaml:6]: image

Comment (Fazziekey) [reply examples/images/diffusion/configs/train_colossalai_teyvat.yaml:6]: It means how to define the loss

Comment (FrankLeeeee) [reply examples/images/diffusion/configs/train_colossalai_teyvat.yaml:6]: I wonder if there is any doc to explain the parameters, which can significantly help the users to understand the config file.

File: examples/images/diffusion/main.py [hunk 4]
```diff
@@ -533,8 +518,8 @@ def on_train_epoch_end(self, trainer, pl_module):
             "If you want to resume training in a new log folder, "
             "use -n/--name in combination with --resume_from_checkpoint"
         )
-    if opt.flash:
-        enable_flash_attention()
+    # if opt.flash:
+    #     enable_flash_attention()
```
[the remark this answers is not in the document]
Comment (Fazziekey) [reply examples/images/diffusion/main.py:522]: deleted, the flash attention is supported in x-formers now

File: examples/images/diffusion/README.md (hunk 1 above)
Comment (feifeibear) [reply examples/images/diffusion/README.md:52]: 0.1.12 is a stable version. you should use the latest release version.

Comment (Fazziekey) [reply examples/images/diffusion/README.md:52]: ok

**Review by feifeibear — APPROVED** [review]

File: examples/images/diffusion/README.md [hunk 5]
```diff
@@ -39,9 +39,17 @@ pip install transformers==4.19.2 diffusers invisible-watermark
...
 
+### install lightning
+```
+git clone https://github.com/1SAA/lightning.git
+git checkout strategy/colossalai
+export PACKAGE_NAME=pytorch
+pip install .
+```
+
 ### Install [Colossal-AI v0.1.10](https://colossalai.org/download/) From Our Official Website
```
Comment (feifeibear) [inline examples/images/diffusion/README.md:50]: Colossal-AI v0.1.12
